docs/user-guide/global-configurations/image-promotion-policy.md

```diff
@@ -0,0 +1,136 @@
+    * **Fail Condition**: Images that fail the condition will not be directed to the target environment.
+
+    **Example**: *`branchName.startsWith('hotfix')`*
+
+{% hint style="info" %}
+Use **View filter criteria** to check the supported variables.
+{% endhint %}  
+
+5. (Optional) If required, you can also introduce an approval flow using the toggle button: **Approval for Image Promotion Policy**. As a result, the [user selecting an image for promotion](#selecting-image-for-promotion) will [require approval](#approving-an-image-promotion-request).
+
+ * **Number of Approvals (1-6)**: Specify the number of approvals required to promote an image. This can vary from one approval (minimum) to six approvals (maximum).
+
+ * **Checkboxes for who can approve**: By default, users who [request an image promotion](#selecting-image-for-promotion) cannot approve their own request. The same is true for those who trigger the build. Whereas, the ones who [approve an image promotion request](#approving-an-image-promotion-request) are barred from deploying the promoted image. However, as a super-admin, you can use the checkboxes to remove such restrictions for better control.
+
+6. Click **Save Changes**.
+
+---
+
+## Applying an Image Promotion Policy
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to apply an image promotion policy.
+{% endhint %}
+
+Here, you can decide the application(s) and environment(s) for which image promotion is allowed. 
+
+1. Go to the **Apply Policy** tab.
+
+2. Use the checkbox to select an application.
+
+3. Click the `Promotion Policy` dropdown within it and choose the policy you wish to apply.
+
+4. A confirmation dialog box would appear. Click **Confirm**.
+
+### Performing Bulk Action
+
+You can also apply a policy to more than one application. Simply use the checkboxes to select the applications. You can do this even if there are many applications spanning multiple pages. You will see a draggable floating widget.
+
+Moreover, there are three filters available to make it easier for you to make the selections easier:
+* Application
+* Environment
+* Policy
+
+---
+
+## Result
+
+### Selecting Image for Promotion
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have build & deploy permission or above (along with access to the application and target environment) to select an image for promotion.
+{% endhint %}
+
+Here, you can promote images to the target environment(s).  
+
+1. Go to the **Build & Deploy** tab of your application.
+
+2. Click the **Promote** button next to the pipeline eligible for image promotion.
+
+3. In the `Select Image` tab, you will see a list of images. However, you can use the **Show Images from** dropdown to decide the image you wish to promote. It can be an image either from the CI pipeline or from any one of the intermediate stages (environments).
```

Comment:
It can be an image either from the CI pipeline or an image that has passed from a particular environment in the workflow (the image must have passed all stages of deployment).

//Can we have a visual representation to drive this point home?